addons/medical_treatment/functions/fnc_placeInBodyBag.sqf

```diff
@@ -16,38 +16,11 @@
  * Public: No
  */
 
-params ["_medic", "_patient"];
-TRACE_1("placeInBodyBag",_patient);
+private _alive = alive _patient;
 
-if ((alive _patient) && {!GVAR(allowBodyBagUnconscious)}) exitWith {
+if (_alive && {!GVAR(allowBodyBagUnconscious)}) exitWith {
     [_medic, "ACE_bodyBag"] call EFUNC(common,addToInventory); // re-add slighly used bodybag?
```

Comment:
```suggestion
    [_medic, "ACE_bodyBag"] call EFUNC(common,addToInventory);
```

Reply:
I figured it would be doubtful that it would ever be implemented, but at the same time, the comment doesn't hurt.